src/js/edu-benefits/components/military-history/RotcHistoryFields.jsx

```diff
@@ -27,7 +27,7 @@ export default class RotcHistoryFields extends React.Component {
             onValueChange={(update) => {this.props.onStateChange('seniorRotcCommissioned', update);}}/>
         <div>
           <div className="input-section">
-            <ErrorableNumberInput required
+            <ErrorableNumberInput
                 additionalClass="usa-input-medium"
                 errorMessage={validateIfDirty(this.props.data.seniorRotc.commissionYear, isValidYear) ? undefined : 'Please enter a valid year'}
```

Comment:
Do you get an invalid year message if you focus and then blur the field without entering anything? I wonder if this needs to be changed to skip isValidYear if the year is blank.